microcosm_flask/audit.py

```diff
@@ -205,6 +223,13 @@ def capture_response(self, response):
             # only capture request body if there is one
             return
 
+        if all((
+            self.options.include_response_body is not True,
+            len(body) >= self.options.include_response_body,
```

Comment:
Just being paranoid, but will this comparison fire in some weird way if `self.options.include_response_body == False`?

Reply:
Nope:

```
>>> 1 >= False
True
>>> 0 >= False
True
>>>
```

If it's false, this evaluates to `True` and we skip, which is the meaning of `incldue_rsponse_body=False` in this case.